plugins/robothor_plugin/configs/debug_objectnav_robothor_rgb_ddppo.py

```diff
@@ -0,0 +1,71 @@
+import os
```

Comment:
If we are including this, should we also include the scripts for generating the dataset to begin with? And if so, should the debug set generation script not just be a flag in that script? We can have this for now, but it might be smart to merge these in the future?

Reply:
Would be nice to have these scripts but I don't necessarily consider it to be a must have. It would be nice to centralize things but the advantage of having different scripts is that the amount of code that has to be read remains somewhat small (anyone can quickly take the debug dataset generation script and change it to create a new dataset if they'd like).